app/controllers/items_controller.rb

```diff
@@ -23,6 +23,28 @@ def new
   def edit
   end
 
+  # GET /items/borrowed
+  # lists all items borrowed by the current user
+  def borrowed_items
+    @items = current_user.lendings.where(completed_at: nil).map(&:item)
+    render :borrowed_items
+  end
+
+  # GET /items/my/borrowed
+  # lists all items of the current user which are currently borrowed
+  def my_borrowed_items
+    # return items which are currently not lendable
+    @items = current_user.items.reject(&:lendable?)
```

Comment:
- optimization: AFAICT `#reject` works app-side; this is probably fine for now, but if there might be huge owners with hundreds or thousands of own items, this check should probably be deferred to the database.

- style: this code returns all items that are not lendable; right now, these are only borrowed items, but for example in the future we might add something like locked/damaged state, which also precludes lendability without implying borrowedness, so it is probably preferable to have a separate `#borrowed?` method and then make `#lendable?` to `!borrowed?`.

- structure: I also think that the lendable? check is something, where Rails scopes (https://guides.rubyonrails.org/active_record_querying.html#scopes) might be a good fit, please check out.